Entities/Materials/Ammunition/MaterialWaterArrow.as

```diff
@@ -9,4 +9,17 @@ void onInit(CBlob@ this)
   this.maxQuantity = 2;
 
   this.getCurrentScript().runFlags |= Script::remove_after_this;
+
+  // Pickup: custom hover area
+  Vec2f[] hoverRect = 
```

Comment:
This is repeated across all arrow material scripts, maybe introduce a `genericArrowHoverRect` constant in a `ArrowCommon.as` script? Or some kind of constants script (e.g. `ArrowConstants.as`) though nothing else in KAG seems to follow such a convention (yet)...

Reply:
I moved the shape into `ArrowCommon.as`